js/pages/tola_management_pages/user/models.js

```diff
@@ -678,13 +690,11 @@ export class CountryStore {
     @observable regions;
     @observable countries;
     @observable _selectedCountryIds;
-    @observable _expandedCountryIds;
 
     constructor(regions, countries) {
         this.regions = regions;
         this.countries = countries;
         this._selectedCountryIds = [];
-        this._expandedCountryIds = new Set([...Object.keys(this.countries).map(id => parseInt(id))]);
         this.nameSort = (objA, objB) => (objA.name.toUpperCase() < objB.name.toUpperCase()) ? -1 : 1;
```

Comment:
these too - looks like you didn't make changes to CountryStore - so maybe we just keep all the CountryStore items and take your changes on the store above it?